docs/spec.md

```diff
@@ -132,6 +139,66 @@ Informs the client that channels are no longer available.
 }
 ```
 
+### Parameter Values
+
+This message is sent as a response to the client's [Get Parameters request](#get-parameters). Only supported if the server declares the `parameters` [capability](#server-info).
+
+#### Fields
+
+- `op`: string `"parameterValues"`
+- `parameters`: array of:
+  - `name`: string, name of the parameter
+  - `value`: number | boolean | string | number[] | boolean[] | string[]
+  - `meta`: object, only given if requested with `includeMetaInfo: true`
+    - `type`: bool | int64 | float64 | string | byte[] | bool[] | int64[] | float64[] | string[]
+
+#### Example
+
+```json
+{
+  "op": "parameterValues",
+  "parameters": [
+    { "name": "/int_param", "value": 2, "meta": { "type": "int64" } },
+    { "name": "/float_param", "value": 1.2, "meta": { "type": "float64" } },
+    { "name": "/string_param", "value": "foo", "meta": { "type": "string" } },
+    {
+      "name": "/node/nested_ints_param",
+      "value": [1, 2, 3],
+      "meta": { "type": "int64[]" }
+    }
+  ]
+}
+```
+
+### Parameter Updates
+
+Informs the client about updates of parameters that the client previously [subscribed](#subscribe-parameter-update) to. Only supported if the server declares the `parameters` [capability](#server-info).
+
+#### Fields
+
+- `op`: string `"parameterUpdates"`
+- `parameters`: array of:
+  - `name`: string
+  - `newValue`: number | boolean | string | number[] | boolean[] | string[]
+  - `oldValue`: number | boolean | string | number[] | boolean[] | string[]
```

Comment:
Why include oldValue?

Reply:
Not necessary, I agree

```suggestion
```

Reply:
Drop `oldValue`, rename `newValue` to `value`